CHANGES.rst

```diff
@@ -543,6 +543,8 @@ New Features
 - ``astropy.tests``
 
   - Added a new Quantity-aware ``assert_quantity_allclose``. [#3273]
```

Comment:
Is this going to be for v1.1.0?  If so this should be moved to the right section.